### PR TITLE
activemq: https, sha256

### PR DESCRIPTION
**Pull request by stevelaskaridis**

### CODE REVIEW

File: Library/Formula/activemq.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Activemq < Formula
-  homepage "http://activemq.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=/activemq/5.11.1/apache-activemq-5.11.1-bin.tar.gz"
-  sha1 "db4310f037af4a430abfdddf18c2df4082a34340"
-
+  homepage "https://activemq.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=/activemq/5.11.1/apache-activemq-5.11.1-bin.tar.gz"
+  sha256 "5ae90f4ea6caa3af7d9f79d1cc55b575dd44170b1451f096494e1a356828d35f"
+
```
Comment (asparagui) [inline Library/Formula/activemq.rb]: Deleting this will pass audit on the bots, thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by stevelaskaridis** [comment]
@asparagui, could you, please, elaborate on your note as I am not sure I understand the part you are referring to ... thanks!

**Comment by stevelaskaridis** [comment]
No worries, found that it was the trailing space, which I couldn't see.
Showing invisible chars and checking the error message in Jenkins made me realize it.

**Pull request closed without merging**